packages/next-server/lib/head.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import sideEffect from './side-effect'
+import HeadManagerContext from './head-context'
 
 class Head extends React.Component {
-  static contextTypes = {
-    headManager: PropTypes.object
-  }
+  static contextType = HeadManagerContext
```

Comment:
We need `react >= 16.6` for this. We could use the `<Consumer/>` instead, but then we'd have to wrap the `<Head/>` as pass the context as a prop. So we'd have two components instead of one.

Reply:
Let's use  contextType and bump the peerDeps to 16.6.